test/unit/rules/lint-no-outlet-outside-routes-test.js

```diff
@@ -24,6 +24,19 @@ generateRuleTests({
         column: 0
       }
     },
+    {
+      template: '{{#outlet}}Route with "components" name{{/outlet}}',
+      meta: {
+        moduleId: 'app/templates/components.hbs',
```

Comment:
Can you double check and confirm this would have failed previously? AFAICT the original code looked for `components/` which would not match this tests module...